server/app/views/applicant/ApplicantProgramSummaryView.java

```diff
@@ -160,11 +160,13 @@ public Content render(Params params) {
 
   /** Renders {@code data} including the question and any existing answer to it. */
   private DivTag renderQuestionSummary(AnswerData data, Messages messages, long applicantId) {
-    DivTag questionPrompt = div(data.questionText()).withClasses("font-semibold");
-    if (!data.applicantQuestion().isOptional()) {
```

Comment:
suggestion (blocking):  Adding the required question indicator after formatting the text (similar to this) might solve the bug with the markdown on required questions that I described in the other comment.

Reply:
smart! i'll try that.

Reply:
I realized the issue was that I was only attaching the required indicator to p tags! I've got a fix I'll push up.